src/pages/imprint.tsx

```diff
@@ -35,11 +35,11 @@ const ImprintPage = ({ data, location }: ImprintPageProps) => {
         location={location}
         noIndex={true}
       />
-      <ContentBlockContainerWithoutHero>
+      <ContentBlockContainer>
         <SectionHeader headline={t('navigation.imprint')} />
         <MarkdownAst htmlAst={data.markdownRemark.htmlAst} />
         <BottomNote>{t('imprint.updated')}</BottomNote>
-      </ContentBlockContainerWithoutHero>
+      </ContentBlockContainer>
```

Comment:
This is the reason I included the layout/container refactoring in this PR. We expected the users to tell if there is a hero or not because the layout did not allow the content to be displayed independent of the hero. I changed that with some default spacing and a proper content block container that counts in the fact, when it's the first child of the layout please don't add to much of spacing.